Automattic/jetpack
# Jetack plugin: refactor connect screen to prevent duplicate

## Proposed changes:

The base connection screens in the Jetpack plugin and My Jetpack package are almost similar. This PR makes the body of the connection screen in the My Jetpack package available and uses that component in the Jetpack plugin.

### Other information:


## Jetpack product discussion


See this comment: https://github.com/Automattic/jetpack/issues/35485#issuecomment-1946836406

## Does this pull request change what data or activity we track or use?


No.

## Testing instructions:






- Spin up a test site with this branch
- Connect the site from these two places and check that everything works as before:
  - `/wp-admin/admin.php?page=jetpack#/connect-user`
  - `/wp-admin/admin.php?page=my-jetpack#/connection`
